checker/build.gradle

```diff
@@ -2,7 +2,7 @@ ext {
     // See instructions for updating jdk8.jar at
     // https://github.com/typetools/annotated-libraries/blob/master/README.md .
     // Set jdkShaHash to 'local' to use a locally-built version of jdk8.jar .
-    jdkShaHash = 'c47e5da9b2b1c8627981c97b75ec0d4cbd7ee052'
+    jdkShaHash = 'cf9185c03d6a63ad28f7dd51615308beeff7a64a'
```

Comment:
A [change](https://github.com/typetools/annotated-libraries/commit/c703b5750e37538731189bdce45ca80b67986f22) was made to https://github.com/typetools/annotated-libraries since you made your [pull request](https://github.com/typetools/annotated-libraries/pull/16) against it, and also I [changed](https://github.com/typetools/checker-framework/commit/0b77004b1b2f999d40e64d7414378d3e1796585f) a library annotation (see explanation below).  Could you please update that pull request and this SHA hash?

Reply:
I have updated the pull request and SHA hash.

Reply:
The [pull request](https://github.com/typetools/annotated-libraries/pull/16) still shows "This branch has conflicts that must be resolved".  Can you please pull master into your branch?  That is the way to update the pull request.  Thanks.

Reply:
@mernst I have removed the conflict. Please review.
Thanks.